`data.azurerm_api_management` - Support  property `tenant_access` (#19422)
hashicorp/terraform-provider-azurerm: website/docs/d/api_management.html.markdown

---

> A `tenant_access` block supports the following:

Reviewer: this is read-only:

```suggestion
A `tenant_access` block exports the following:
```

Author: Thanks for you feedback. Fixed.

> * `enabled` - Should the access to the management API be enabled?

Reviewer: this is read-only:

```suggestion
* `enabled` - Is access to the Management API enabled (presumably "for this Tenant")?
```

Author: Thanks for you feedback. Fixed.

> * `tenant_id` - The identifier for the tenant access information contract.

Reviewer: presumably:

```suggestion
* `tenant_id` - The ID of the Tenant which has access to this API Management instance.
```

?

Author: Thanks for you feedback. Fixed.
